src/nnue/network.h

```diff
@@ -40,6 +41,7 @@ enum class EmbeddedNNUEType {
     SMALL,
 };
 
+typedef std::tuple<Value, Value> NetworkOutput;
```

Comment:
please use a type alias instead of typedef

Reply:
honestly is it even worthwhile to rename the tuple at all